backend/fms_core/services/utils.py

```diff
@@ -0,0 +1,19 @@
+def fit_string_with_ellipses(string: str, max_length: int, ellipses = "...") -> str:
+    if max_length < 0:
+        raise Exception(f"The max_length '{max_length}' must be a positive integer")
+    if max_length < len(ellipses):
```

Comment:
i think we are using the french term ... I would still use the singular across the code in any case.
ellipsis [noun] the deliberate omission of a word or words from a sentence that do not prevent the meaning from being understood.